commentActivity.js

```diff
@@ -14,5 +14,5 @@ Med:
 Mer:
 Mor:
 Pal:
-Ram:
+Ram: What is your favorite album?
```

Comment:
Hi carlos. That is a super tough question. I am a big music fan. I listen to a lot of music and play a lot of instruments. If i had to narrow it down to one i would either say Seargeant Peppers Lonely Album, or pink floyds dark side of the moon.